src/demo/pages/BlockchainTx.tsx

```diff
@@ -111,133 +122,100 @@ export default function BlockchainTx() {
-          asset.
-        </TP>
-        <TP className="list">
-          Other examples of how to use credentials with Tonomy ID include:
-          <ul>
-            <li>
-              <TImage src={Handshake} alt="ticket" className="listIcon" />
-              employment contracts
-            </li>
-            <li>
-              <TImage src={Cruise} alt="ticket" className="listIcon" />
-              shipping and logistics
-            </li>
-            <li>
-              <TImage src={Money} alt="ticket" className="listIcon" />
-              events payments and payment metadata
-            </li>
-            <li>
-              <TImage src={Vote} alt="ticket" className="listIcon" />
-              votes event
-            </li>
-            <li>
-              <TImage src={Ticket} alt="ticket" className="listIcon" />
-              tickets and much more!
-            </li>
-          </ul>
-        </TP>
-
-        {transactionState !== "purchased" ? (
-          <BoxContainer className="boxStyle1">
-            <TH2>#85456</TH2>
-            <div className="nftImageColumn">
-              <div className="imgConatiner">
-                <TImage
-                  src={connectionImage}
-                  alt="Connecting Phone-PC"
-                  className="nftImage"
-                />
-                <div className="paddingTop">
-                  <TP>Sale ID: #197387654094</TP>
-                  <TP>Owned by: 948361751KIHF </TP>
-                  <TP>Created by: ‘federation’</TP>
-                  <TP>Blockchain: Tonomy Demo Network</TP>
-                  <div className="marginTop">
-                    <h4 style={{ textAlign: "left" }}>Price</h4>
-                    <TP>17.12 SYS (€1,950.53 DEMO ONLY )</TP>
-                  </div>
-                  <TButton
-                    disabled={transactionState === "loading"}
-                    className="tbuttonstyle"
-                    onClick={() => onBuy()}
-                  >
-                    BUY
-                  </TButton>
-                </div>
-              </div>
+    <>
+      <HeaderContainer>
+        <HeaderPictureContainer></HeaderPictureContainer>
+        <FeatureNameLabel>Feature Name: Sign Transaction</FeatureNameLabel>
+
+        <HowToUseLabel>How to use :</HowToUseLabel>
+        <HeaderTonomy>Tonomy ID</HeaderTonomy>
+        {/* <HeaderTonomyID>Tonomy ID</HeaderTonomyID> */}
+        <HeaderDescription>
+          Tonomy ID utilizes a digital signatures and a distributed transaction
+          protocol to safeguard your transactions and digital assets from
+          unauthorized access or tampering.
+        </HeaderDescription>
+
+        <Link href="/Test">learn about the Antelope blockchain protocol </Link>
+        <br />
+        <br />
+        <Button variant="outlined" size="large">
+          Enter Demo
+        </Button>
+      </HeaderContainer>
+      <MainDescription>
```

Comment:
it's better to use the paragragh and heading tags instead of such tags and use component level css

Reply:
Done